src/Donors/Migrations/addPhoneColumn.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Give\Donors\Migrations;
+
+use Give\Framework\Database\Exceptions\DatabaseQueryException;
+use Give\Framework\Migrations\Contracts\Migration;
+use Give\Framework\Migrations\Exceptions\DatabaseMigrationException;
+
+/**
+ * @unreleased
+ */
+class addPhoneColumn extends Migration
+{
+    /**
+     * @unreleased
+     *
+     * @throws DatabaseMigrationException
+     */
+    public function run()
+    {
+        global $wpdb;
+
+        $donorsTableName = "{$wpdb->prefix}give_donors";
+
+        try {
+            maybe_add_column(
```

Comment:
I think this is the first time i've seen `maybe_add_column` used in our migrations haha! I actually think it's a good use case to check if the column already exists 👍